aws/services/ds/resource.ftl

```diff
@@ -0,0 +1,50 @@
+[#ftl]
+
+[#assign DIRECTORY_OUTPUT_MAPPINGS =
+    {
+        REFERENCE_ATTRIBUTE_TYPE : {
+            "UseRef" : true
+        }
+    }
+]
+[@addOutputMapping
+    provider=AWS_PROVIDER
+    resourceType=AWS_DIRECTORY_RESOURCE_TYPE
+    mappings=DIRECTORY_OUTPUT_MAPPINGS
+/]
+
+[#macro createDSInstance id name
+    type
+    size
```

Comment:
Probably good to set the size with a default of "" since its not going to be required for everything right?

Reply:
When I collapsed edition and size, it becomes mandatory (or at least defaulted to Small or Large)